ctools/defragment_sharded_collection.py

```diff
@@ -666,9 +666,16 @@ def get_chunk_size_or_0(ch):
 
             # avoid moving larger chunks
             center_size_kb = await get_chunk_size(c)
+
             # Use < 0.6 so that we do not move chunks which were split before
-            if center_size_kb > target_chunk_size_kb * args.small_chunk_frac:
+            has_local_est_size = True if 'defrag_collection_est_size' in c else False
```

Comment:
Be aware that 'defrag_collection_est_size' is always set by https://github.com/kaloianm/workscripts/blob/f9edb33c274cb720dd287c15f9ee796e5dfdb9af/ctools/defragment_sharded_collection.py#L668 via https://github.com/kaloianm/workscripts/blob/f9edb33c274cb720dd287c15f9ee796e5dfdb9af/ctools/defragment_sharded_collection.py#L615

so this line needs to move before 
`enter_size_kb = await get_chunk_size(c)`